src/app/fyle/personal-cards-matched-expenses/expense-preview/expense-preview.component.ts

```
@@ -62,14 +60,9 @@ export class ExpensePreviewComponent implements OnInit {
     this.loading = true;
-    this.launchDarklyService
-      .getVariation('personal_cards_platform', false)
-      .pipe(
-        switchMap((usePlatformApi) =>
-          this.personalCardsService.matchExpense(this.expenseId, this.cardTxnId, usePlatformApi)
-        ),
-        finalize(() => (this.loading = false))
-      )
+    this.personalCardsService
+      .matchExpense(this.expenseId, this.cardTxnId)
+      .pipe(finalize(() => (this.loading = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)